setup.py

```diff
@@ -44,6 +44,7 @@ def read(fname):
                             "ephem",
                             "h5py",
                             "pandas",
+                            "paramsurvey[ray]",
```

Comment:
You're installing `paramsurvey[ray]` which means everyone's laptop is
going to get a not-needed Ray installation. I'm not sure how
diligent the Ray team is at making macOS wheels.

I'd recommend installing plain `paramsurvey` and depend on this bit of
code already in paramsurvey to tell cluster/cloud users that they need
to install `paramsurvey[ray]`:

```     
        raise ValueError('Import of ray module failed, did you "pip install paramsurvey[ray]"?')
```